router/router.go

```diff
@@ -918,9 +918,12 @@ func (worker *workerT) handleWorkerDestinationJobs(ctx context.Context) {
 				//Only one job is marked failed and the rest are marked waiting
 				//Job order logic requires that at any point of time, we should have only one failed job per user
 				//This is introduced to ensure the above statement
-				resp := fmt.Sprintf(`{"blocking_id":"%d", "user_id":"%s", "moreinfo": "attempted to send in a batch"}`, prevFailedJobID, destinationJobMetadata.UserID)
+				resp := misc.UpdateJSONWithNewKeyVal([]byte(`{}`), "blocking_id", prevFailedJobID)
```

Comment:
@chandumlg  looks a bit complicated, why didn't you prefer marshalling a map instead?

```
resp, _ := json.Marshal(map[string]interface{}{
	"blocking_id": prevFailedJobID,
	"user_id":     destinationJobMetadata.UserID,
	"moreinfo":    "attempted to send in a batch",
})
```